### DISCUSSION TITLE
In mobile resolution, navbar items are not going inside the left sidebar

### DISCUSSION THREAD

**Discussion by Karan-S-dev** [facebook/docusaurus#8208] [Q&A] [answered]
When I shift to mobile resolution, the navbar items are going inside the left sidebar. But there is a login button, which is custom added. Swizzled navbar component and added this login button. It is also going inside the left sidebar but also present in the navbar. 

You can go and have a look inside the website. You will get the context of my question.

When I switch to mobile resolution, I want all my navbar items, to go inside the left side bar and nothing should appear in the navbar
except search bar.

**Comment by slorber** [comment, marked as answer]
If you create custom navbar items such as this login button, it is your responsibility to add a CSS mediaquery to hide it for smaller viewports. Docusaurus can't know what you want to do exactly for you. Some users may want the login button to remain visible on mobile for example

Maybe add the class `navbar__item` to this login button 🤷‍♂️

**Reply by Karan-S-dev** [reply]
But when I add this, it is also getting disappeared from the left sidebar in mobile resolution. It needs to be inside the sidebar.

**Reply by slorber** [reply]
Then you have to swizzle things unfortunately or use more advanced selectors so that it only hides in the desktop navbar

It's your UI item, your responsibility
